src/chocolatey/infrastructure.app/commands/ChocolateyApiKeyCommand.cs

```diff
@@ -100,9 +123,10 @@ public virtual void help_message(ChocolateyConfiguration configuration)
             "chocolatey".Log().Info(@"
     choco apikey
     choco apikey -s https://somewhere/out/there
-    choco apikey -s=""https://somewhere/out/there/"" -k=""value""
-    choco apikey -s ""https://push.chocolatey.org/"" -k=""123-123123-123""
-    choco apikey -s ""http://internal_nexus"" -k=""user:password""
+    choco apikey add -s=""https://somewhere/out/there/"" -k=""value""

```

Comment:
Can we add an example of using `choco apikey list` here?

Reply:
Added this one too